server/models/receipt.js

```diff
@@ -17,6 +17,7 @@ const Receipt = mongoose.model(
         },
         date: {
             type: Date,
+            default: Date.now,
```

Comment:
this is also for the next task, but this is added so all receipts added will default have the date that they create the receipt.